frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -992,6 +1116,8 @@ impl<T: Config> Pallet<T> {
 		// change phase
 		<CurrentPhase<T>>::put(Phase::Off);
 
+		// TODO: clean signed submissions: test case: call `elect` in the middle of the signed phase.
```

Comment:
Unresolved TODO

Reply:
What is necessary here that isn't covered by the [`early_termination`](https://github.com/paritytech/substrate/blob/d5fe6713319dba94b00d8e3b1461c0a0be108767/frame/election-provider-multi-phase/src/lib.rs#L1577) test?

Reply:
I think 75ddba933 clears this.

Reply:
Oh, I see. Have a failing test case now; will push once I have it passing.

Reply:
abae60234